From cd0d77f12981383f7150624276e479a6799db1ba Mon Sep 17 00:00:00 2001
From: Samuel Ortiz <sameo@linux.intel.com>
Date: Mon, 8 Sep 2014 15:51:12 +0200
Subject: [PATCH 2/2] iio: bmm050: Make it less chatty

The "data not updata" message can be sent 100+ times per second
when userspace uses that device.

Issue: IRDA-710
Change-Id: I859d3ac67fd381a33658648ce264dab99905798d
Signed-off-by: Samuel Ortiz <sameo@linux.intel.com>
---
 drivers/iio/magnetometer/bmm050_driver.c | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/drivers/iio/magnetometer/bmm050_driver.c b/drivers/iio/magnetometer/bmm050_driver.c
index 7572613..332aaf8 100644
--- a/drivers/iio/magnetometer/bmm050_driver.c
+++ b/drivers/iio/magnetometer/bmm050_driver.c
@@ -984,21 +984,21 @@ static int bmm_read_axis_data(struct iio_dev *indio_dev,
 	case BMM_SCAN_MAG_X:
 		/* x aixs data */
 		if (!client_data->iio_mdata.value_x_valid)
-			dev_err(&client->dev, "x data not updata!\n");
+			dev_dbg(&client->dev, "x data not updata!\n");
 		raw_data->data = client_data->value.datax;
 		client_data->iio_mdata.value_x_valid = VALUE_INVALID;
 		break;
 	case BMM_SCAN_MAG_Y:
 		/* y aixs data */
 		if (!client_data->iio_mdata.value_y_valid)
-			dev_err(&client->dev, "y data not updata!\n");
+			dev_dbg(&client->dev, "y data not updata!\n");
 		raw_data->data = client_data->value.datay;
 		client_data->iio_mdata.value_y_valid = VALUE_INVALID;
 		break;
 	case BMM_SCAN_MAG_Z:
 		/* z aixs data */
 		if (!client_data->iio_mdata.value_z_valid)
-			dev_err(&client->dev, "z data not updata!\n");
+			dev_dbg(&client->dev, "z data not updata!\n");
 		raw_data->data = client_data->value.dataz;
 		client_data->iio_mdata.value_z_valid = VALUE_INVALID;
 		break;
@@ -1168,7 +1168,7 @@ static int bmm_restore_hw_cfg(struct i2c_client *client)
 			&client_data->rept_z, 1);
 	mdelay(BMM_I2C_WRITE_DELAY_TIME);
 	err = bmm_i2c_read(client, BMM_REG_NAME(NO_REPETITIONS_Z), &value, 1);
-	dev_info(&client->dev, "BMM_NO_REPETITIONS_Z: %02x", value);
+	dev_dbg(&client->dev, "BMM_NO_REPETITIONS_Z: %02x", value);
 	mutex_unlock(&client_data->mutex_rept_z);
 
 	mutex_lock(&client_data->mutex_op_mode);
-- 
1.8.3.2

